aom.yaml

```diff
@@ -0,0 +1,19 @@
+name: aom
+buildsystem: cmake-ninja
+builddir: true
+config-opts:
+  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+  - -DBUILD_SHARED_LIBS=ON
+  - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+build-options:
+  arch:
+    arm:
+      config-opts:
+        - -DAOM_TARGET_CPU=generic
+sources:
+  - type: git
+    url: https://aomedia.googlesource.com/aom
+    branch: v1.0.0
+    commit: d14c5bb4f336ef1842046089849dee4a301fbbf0
+post-install:
+  - install -p -D -m 0644 "../LICENSE" "../PATENTS" -t "${FLATPAK_DEST}/share/licenses/aom/";
```

Comment:
License are not exported, shouldn't be installed.

Reply:
> License are not exported, shouldn't be installed.

Encouraging breaking the license should not take place here.